tests/test_client.py

```
@@ -22,7 +22,11 @@
 from llm_api._types import Omit
 from llm_api._utils import asyncify
 from llm_api._models import BaseModel, FinalRequestOptions
-from llm_api._exceptions import APIStatusError, APITimeoutError, APIResponseValidationError
+from llm_api._exceptions import (
+    APIStatusError,
+    APITimeoutError,
+    APIResponseValidationError,
```

Ruff (0.13.3)

Redefinition of unused `APIResponseValidationError` from line 21

(F811)